specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ServerAzureADAdministrators.json

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
In the future, please instantiate the PR by making the first commit be the unmodified prior (reviewed) version, that is the most direct ancestor of the new changes. That enables diff view in the PR